pandas/tests/groupby/test_groupby.py

```diff
@@ -1994,6 +1994,19 @@ def test_bool_aggs_dup_column_labels(bool_agg_func):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize("bool_agg_func", ["any", "all"])
```

Comment:
these should be in test_function.py, next to / part of the test_groupby_bool_aggs which fully parametrizes these.

Reply:
moved